platform/ui/src/components/studyList/StudyList.js

```diff
@@ -16,6 +16,69 @@ import { withTranslation } from '../../utils/LanguageProvider';
 const today = moment();
 const lastWeek = moment().subtract(7, 'day');
 const lastMonth = moment().subtract(1, 'month');
+function getPaginationFragment(
+  props,
+  searchData,
+  nextPageCb,
+  prevPageCb,
+  changeRowsPerPageCb
+) {
+  return (
+    <PaginationArea
+      pageOptions={props.pageOptions}
+      currentPage={searchData.currentPage}
+      nextPageFunc={nextPageCb}
+      prevPageFunc={prevPageCb}
+      onRowsPerPageChange={changeRowsPerPageCb}
+      rowsPerPage={searchData.rowsPerPage}
+      recordCount={props.studies.length}
+    />
+  );
+}
+
+function getTableMeta(props) {
+  return {
+    patientName: {
+      displayText: props.t('PatientName'),
+      sort: 0,
+    },
+    patientId: {
+      displayText: props.t('MRN'),
+      sort: 0,
+    },
+    accessionNumber: {
+      displayText: props.t('AccessionNumber'),
+      sort: 0,
+    },
+    studyDate: {
+      displayText: props.t('StudyDate'),
+      inputType: 'date-range',
+      sort: 0,
+    },
+    modalities: {
+      displayText: props.t('Modality'),
+      sort: 0,
+    },
+    studyDescription: {
+      displayText: props.t('StudyDescription'),
+      sort: 0,
+    },
+  };
+}
+
+function getNoListFragment(studies, error, loading) {
+  if (loading) {
+    return (
+      <div className="loading">
+        <StudyListLoadingText />
+      </div>
+    );
+  } else if (error) {
+    return <div className="notFound">There was an error fetching studies</div>;
```

Comment:
Can we wrap these in translation strings (this.props.t()) so they can be internationalized?